Description: Disable reference.md "Items_and_attributes_14" doctest, hanging on armhf
 This passes on i386 amd64 arm64
 .
 See https://gist.github.com/infinity0/2f5452d4ee9deb2f1bacb8b39c987c9c
 for thread stacktrace during the hang; eddyb from #rustc suggests this might
 be an LLVM issue. Unsure if this is Debian-specific; we should re-enable this
 test again when we move to LLVM-3.8 (what Rust upstream is using) and if it
 still occurs then file a bug upstream.
Author: Ximin Luo <infinity0@debian.org>
Bug: TODO pending LLVM-3.8 switch
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/doc/reference.md
+++ b/src/doc/reference.md
@@ -1092,7 +1092,7 @@
 in the same way as any other Rust function, except that they have the `extern`
 modifier.
 
-```
+```{.ignore}
 // Declares an extern fn, the ABI defaults to "C"
 extern fn new_i32() -> i32 { 0 }
 
